lib/ansible/modules/windows/win_scheduled_task.ps1

```diff
@@ -547,48 +626,48 @@ Function Compare-Triggers($task_definition) {
     $map = @{
```

Comment:
The line above this has a TODO about solving repetition, because you have done this are you able to remove it?